runtime/doc/various.txt

```diff
@@ -224,10 +224,17 @@ g8			Print the hex values of the bytes used in the
 
 						  *:term* *:terminal*
 :term[inal][!] {cmd}	Spawns {command} using the current value of 'shell'
-			in a new terminal buffer.  This is equivalent to: >
+			and 'shellcmdflag' in a new terminal buffer.  This is
+			equivalent to: >
 
-			      :enew | call termopen('{cmd}') | startinsert
+			      :enew
+			      :call termopen([&sh, &shcf, '{cmd}'],
+			                    \{'name':'{cmd}'})
+			      :startinsert
 <
+			If no {command} is given, 'shellcmdflag' will not be
```

Comment:
This happens when `{cmd}` is not specified, so it's equivalent to
```vim
enew
call termopen(&shell, {'name' : &shell)
startinsert
```
but after looking `ex_terminal` over, it doesn't look like I correctly do that.  Now fixed.